db/migrate/20131002164449_create_automatic_exploitation_match_sets.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAutomaticExploitationMatchSets < ActiveRecord::Migration
+  def change
+    create_table :automatic_exploitation_match_sets do |t|
+      t.integer :workspace_id
+      t.integer :user_id
```

Comment:
You can use `t.references :user, null: false` here to (1) use the foreign key API and (2) ensure there is always a user.